app/(api)/_datalib/collections/findItem.ts

```diff
@@ -0,0 +1,33 @@
+import { getClient } from '@utils/mongodb/mongoClient.mjs';
+import { NotFoundError } from '@utils/response/Errors';
+
+export async function findCollectionItems(collection: never, query = null) {
```

Comment:
what is `collection: never`? I think collection should come in as a string. Query should come in as an object and have default value of `{}`, not null.